scrypto/src/resource/proof.rs

```diff
@@ -366,7 +387,7 @@ impl ValidatedProof {
     ///
     /// # Panics
     /// Panics if this is not a singleton bucket
-    pub fn non_fungible_local_id(&self) -> NonFungibleLocalId {
+    fn non_fungible_local_id(&self) -> NonFungibleLocalId {
```

Comment:
I feel that these RustDocs by default should be on the trait? I'm not sure if you can always see them on an impl?

Reply:
And ditto to all these impls actually.

Reply:
Thought about that. Feel like we need a round of Rustdoc clean-up after all scrypto changes are in.